xapi.opam

```diff
@@ -10,38 +10,52 @@ build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
 
 depends: [
   "jbuilder" {build & >= "1.0+beta11"}
+
   "alcotest"
   "cdrom"
-  "ctypes" {>= "0.4"}
-  "ctypes-foreign" {>= "0.4"}
-  "fd-send-recv"
+  "ctypes"
+  "ctypes-foreign"
+  "ezxenstore"
+  "http-svr"
   "message-switch-unix"
   "mtime"
+  "ocaml-migrate-parsetree"
   "ounit"
-  "rpc"
+  "pciutil"
+  "ppx_deriving_rpc"
+  "ppx_sexp_conv"
+  "rpclib"
+  "rrdd-plugin"
+  "sexpr"
   "sha"
-  "ssl"
   "stdext"
+  "stunnel"
+  "tar"
   "tar-unix"
-  "vhd-format"
+  "base-threads"
+  "base-unix"
+  "uuid"
   "x509"
-  "xapi-cli-protocol"
   "xapi-client"
+  "xapi-cli-protocol"
   "xapi-consts"
   "xapi-database"
   "xapi-datamodel"
-  "xapi-forkexecd"
-  "xapi-idl"
-  "xapi-inventory"
-  "xapi-libs-transitional"
   "xapi-netdev"
-  "xapi-rrdd-plugin"
+  "xapi-stdext-date"
+  "xapi-stdext-monadic"
+  "xapi-stdext-pervasives"
+  "xapi-stdext-std"
+  "xapi-stdext-threads"
+  "xapi-stdext-unix"
   "xapi-tapctl"
   "xapi-test-utils"
   "xapi-types"
   "xapi-xenopsd"
+  "xcp"
+  "xcp-inventory"
   "xenctrl"
-  "xenstore"
+  "xml-light2"
```

Comment:
This seems to come from perftest, but this is what the dune file says, so updating it in opam is correct, we can look at pruning the dependencies from dune later if needed.